src/commands/createStaticWebApp/ApiLocationStep.ts

```diff
@@ -24,7 +24,6 @@ export class ApiLocationStep extends AzureWizardPromptStep<IStaticWebAppWizardCo
         return !wizardContext.apiLocation;
     }
 
-    // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
I don't see this line of code removed from Eric's commit. Do you think it might be because we might use it again later?

Reply:
Eric could have deleted all instances of this line since the rule was turned off but I'm assuming it was just a time saver for him not to. It doesn't hurt anything to keep these exceptions around but they're also no longer required

Reply:
Yeah I was annoyed with the rule when authoring new code, so I turned it off. I wanted to save time and I didn't want to clutter the diff of that PR. If I was doing a PR to specifically remove the rule, I would do exactly what Will did.